tests/test_parser.py

```diff
@@ -411,13 +411,29 @@ def test_git_renaming(self):
         with codecs.open(file_path, 'r', encoding='utf-8') as diff_file:
             res = PatchSet(diff_file)
 
-        self.assertEqual(len(res), 1)
+        self.assertEqual(len(res), 4)
 
         patch = res[0]
         self.assertTrue(patch.is_rename)
+        self.assertEqual(patch.added, 0)
+        self.assertEqual(patch.removed, 0)
+
+        patch = res[1]
+        self.assertFalse(patch.is_rename)
         self.assertEqual(patch.added, 1)
         self.assertEqual(patch.removed, 1)
-        self.assertEqual(len(res.modified_files), 1)
+
+        patch = res[2]
+        self.assertTrue(patch.is_rename)
+        self.assertEqual(patch.added, 0)
+        self.assertEqual(patch.removed, 0)
+
+        patch = res[3]
+        self.assertFalse(patch.is_rename)
+        self.assertEqual(patch.added, 1)
+        self.assertEqual(patch.removed, 1)
+
+        self.assertEqual(len(res.modified_files), 4)
```

Comment:
Hmm... shouldn't this be 2? Also I would expect we have 2 patch entries in `res`, right?

Reply:
I was wondering about this.
Currently the parser treats the renames as separate patches.

This behaviour works for my purposes, but I don't know what the real behaviour of the library should actually be.